src/Germany/Mews.Fiscalizations.Germany/RequestCreator.cs

```diff
@@ -74,6 +73,23 @@ internal static Dto.AuthorizationTokenRequest CreateAuthorizationToken(ApiKey ap
             };
         }
 
+        internal static Dto.AdminLoginRequest CreateAdminLoginRequest(string adminPin)
```

Comment:
I wouldn't prefix this with Create. Just `AdminLoginRequest`. All the other `Create` words here mean that we are actually creating something. But you're not creating a login. A login isn't an entity.